### DISCUSSION TITLE
update and auto deployments

### DISCUSSION THREAD

**Discussion by tomsvogel** [coollabsio/coolify#1011] [General]
is it just me or do you also have the problem that coolify won't update anymore and the auto deployments won't trigger anymore. Are there any possibilities what I can do.

<img width="306" alt="image" src="https://user-images.githubusercontent.com/2229728/228240079-eb178a5c-96b9-4ef7-be28-901c66ea1b5a.png">
<img width="56" alt="image" src="https://user-images.githubusercontent.com/2229728/228240435-b70288a3-7fd2-449f-a1b4-e1279e92cb5f.png">

**Comment by RayBB** [comment]
@tomsvogel is this still an issue for you or can you close it?
